widget_driver/lib/src/utils/runtime_environment_info.dart

```diff
@@ -0,0 +1,18 @@
+import 'platform_environment.dart';
+
+/// A helper class which gives you info about
+/// the environment for the current runtime.
+class RuntimeEnvironmentInfo {
+  static const String _flutterTestEnvKey = 'FLUTTER_TEST';
+
+  final PlatformEnvironment _platformEnvironment;
+
+  RuntimeEnvironmentInfo({
```

Comment:
```suggestion
  const RuntimeEnvironmentInfo({
```

Reply:
I tired it and I currently cannot make this a const constructor.
Since the `RuntimeEnvironmentInfo` depends on the `PlatformEnvironment` in its constructor.. and the `PlatformEnvironment` is not a const object. 
And I cannot make `PlatformEnvironment` a const object since it depends on the `flutter/foundation.dart` since it needs a web-config variable from there.  and that variable is a variable and not a const.. 

Summary: I cannot make this a const..